addons/hr_work_entry/views/hr_work_entry_views.xml

```diff
@@ -14,10 +14,7 @@
         <field name="name">Work Entry</field>
         <field name="res_model">hr.work.entry</field>
         <field name="view_mode">calendar,tree,form,pivot</field>
-        <field name="context">{
-            'search_default_current_month': True,
-            'search_default_work_entries_error': True
-        }</field>
+        <field name="context">{'search_default_work_entries_error': True}</field>
```

Comment:
It's better to leave to leave the filter on a separate line, that way it's more clear what the changes actually are

Reply:
ok sure I'll put it back the way it was then!